infra/blueprint-test/pkg/bq/bq.go

```diff
@@ -76,6 +77,24 @@ func newCmdConfig(opts ...cmdOption) (*CmdCfg, error) {
 	return gOpts, nil
 }
 
+// initBq checks for a local .bigqueryrc file and creates an empty one if not to avoid forced bigquery initialization, which doesn't output valid json.
+func initBq() (error) {
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		return err
+	}
+	fileName := homeDir + "/.bigqueryrc"
+	 _ , err = os.Stat(fileName)
+	if os.IsNotExist(err) {
```

Comment:
nit: I think we could keep this more left aligned like 
```go
if !os.IsNotExist(err) { return nil }
file, err ...
```

Reply:
Ack